### DISCUSSION TITLE
MacOS Universal?

### DISCUSSION THREAD

**Discussion by Max0nGitHub** [Xpl0itU/WiiUDownloader#108] [Q&A]
Why is it called "Universal" if it only works on MacOS 12 or newer and not on any older versions of MacOS? It would be helpful to see this specified on the README as it is misleading. Does any one know a work around other than than using a different OS? I wanted to try this on an old 2010 MBP running 10.11.6.

**Comment by Xpl0itU** [comment]
It's called universal because it works on both Intel Macs and Apple Silicon Macs
